### DISCUSSION TITLE
how to see the test that failed

### DISCUSSION THREAD

**Discussion by slevinnnn** [codewars/codewars.com#3085] [Q&A]
hi everyone iam working on a codewar problem and i passed 114 and failed 1 test, but the page doesnt show me what the problem is , so i dont know what to fix, can anyone tell how can i see what the fail test is for any codewar proyect?
PD: my first ever github post hehe.

**Comment by ejini6969** [comment]
This is not the *right* place to discuss such matters, preferably the discourse of the trained kata is a more liable option. You should [read this first on how to troubleshoot solutions of a kata](https://docs.codewars.com/training/training-example/)
